docker/pulsar-all/pom.xml

```diff
@@ -150,46 +150,8 @@
                           <tag>${project.version}</tag>
                         </tags>
                       </build>
-                    </image>
-                  </images>
-                </configuration>
-              </execution>
-              <execution>
-                <id>push-latest</id>
-                <goals>
-                  <goal>push</goal>
-                </goals>
-                <configuration>
-                  <images>
-                    <image>
-                      <name>${docker.organization}/pulsar-all</name>
-                      <build>
-                        <contextDir>${project.basedir}</contextDir>
-                        <tags>
-                          <tag>latest</tag>
-                        </tags>
-                      </build>
-                    </image>
-                  </images>
-                </configuration>
-              </execution>
-              <execution>
-                <id>add-no-repo</id>
-                <phase>package</phase>
-                <goals>
-                  <goal>build</goal>
-                </goals>
-                <configuration>
-                  <images>
-                    <image>
-                      <name>pulsar-all</name>
-                      <build>
-                        <contextDir>${project.basedir}</contextDir>
-                        <tags>
-                          <tag>latest</tag>
-                          <tag>${project.version}</tag>
-                        </tags>
-                      </build>
+                      <tagName>latest</tagName>
+                      <repo>pulsar-all</repo>
```

Comment:
Perhaps you should add this config to a separate execution with `tag` goal?

Reply:
You may add an execution block under `<goal>build</goal>`. Or do you verify this patch locally? When I read the doc of dmp, it shows that `tagName` is a top-level config instead of inside `image`.

Reply:
Fixed! We can use the CI to verify this.

Reply:
Yep. Could you add a step to verify the build & tag results are correct?